src/tools/tool-options.h

```diff
@@ -239,6 +242,15 @@ struct ToolOptions : public Options {
     passOptions.arguments[key] = value;
   }
 
+  void parse(int argc, const char* argv[]) override {
+    Options::parse(argc, argv);
+    if (enabledFeatures & FeatureSet::BulkMemory) {
+      // Enable this subfeature for backward compatibility.
+      enabledFeatures.setBulkMemoryOpt();
+      disabledFeatures.setBulkMemoryOpt(false);
+    }
```

Comment:
If we'd rather not do this as part of parsing, we could also implement this feature implication in the parsing of the `--enable-bulk-memory` and `--disable-bulk-memory` flags themselves. Giving `addFeature` an optional parameter that is a list of implied features would be nice.

Reply:
Sounds good, done.